supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -176,52 +174,37 @@ class PipelinesApiAsyncBackendJobExecutionActor(standardParams: StandardAsyncExe
   import mouse.all._
 
   private def generateGcsLocalizationScript(inputs: List[PipelinesApiInput],
-                                            referenceFileToDiskImageMappingOpt: Option[Map[String, PipelinesApiReferenceFilesDisk]])
+                                            referenceInputsToMountedPathsOpt: Option[Map[PipelinesApiInput, String]])
                                            (implicit gcsTransferConfiguration: GcsTransferConfiguration): String = {
-    val optionName = WorkflowOptions.UseReferenceDisks.name
-    val useReferenceDisks = workflowDescriptor.workflowOptions.getBoolean(optionName) match {
-      case Success(value) => value
-      case Failure(OptionNotFoundException(_)) => false
-      case Failure(f) =>
-        // Should not happen, this case should have been screened for and fast-failed during workflow materialization.
-        log.error(f, s"Programmer error: unexpected failure attempting to read value for workflow option '$optionName' as a Boolean")
-        false
-    }
-
     // Generate a mapping of reference inputs to their mounted paths and a section of the localization script to
     // "faux localize" these reference inputs with symlinks to their locations on mounted reference disks.
-    def generateReferenceInputsAndLocalizationScript: (Option[Map[PipelinesApiInput, String]], String) = {
-      val referenceInputsToMountedPathsOpt: Option[Map[PipelinesApiInput, String]] =
-        referenceFileToDiskImageMappingOpt.map(getReferenceInputsToMountedPathMappings(_, inputs))
-
-      val referenceFilesLocalizationScript = {
-        val symlinkCreationCommandsOpt = referenceInputsToMountedPathsOpt map { referenceInputsToMountedPaths =>
-          referenceInputsToMountedPaths map {
-            case (input, absolutePathOnRefDisk) =>
-              s"mkdir -p ${input.containerPath.parent.pathAsString} && ln -s $absolutePathOnRefDisk ${input.containerPath.pathAsString}"
-          }
+    val referenceFilesLocalizationScript = {
```

Comment:
It seems surprising the lines below are marked as not being tested, unless Centaur coverage is not being included for some reason...